lms/djangoapps/grades/tasks.py

```diff
@@ -16,8 +17,8 @@
 from .new.subsection_grade import SubsectionGradeFactory
 
 
-@task()
-def recalculate_subsection_grade(user_id, course_id, usage_id):
+@task(bind=True, default_retry_delay=30)
```

Comment:
Cleaning up `block_structure` to properly retry here: https://github.com/edx/edx-platform/pull/13773